src/components/common/Slide.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable max-len */
+
+export function Slider({slideIndex, index, imgSrc, imgId, imgAlt, imgStyle, gridSlider}) {
+  return (
+          <div
+            style={imgStyle}
+            id={imgId}
+            className={`${
+              slideIndex === (gridSlider ? (index+1) : index + 1) ? "slide active-anim" : "slide"
```

Comment:
I think this line is missing something, the ternary in the condition always returns the same value `index + 1`

```suggestion
              slideIndex === index + 1 ? "slide active-anim" : "slide"
```

Reply:
Solved